src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -53,7 +55,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                 {
                     // what do we do if there's no master?
                     TargetBranch = x.FirstOrDefault(b => b.Name == "master");
-                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head.FriendlyName);
+
+                    // 'repo.Head' will be null in unit tests.
+                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head?.FriendlyName);
```

Comment:
Yeah, I wasn't happy about that either. Problem is that the `repo` comes from `GitService.GitServiceHelper.GetRepo` which is a static method and can't be mocked. I started refactoring to allow this but aborted as I kinda wanted to keep the PR to the point. I can go back to it if you'd like though.